src/openforms/registrations/contrib/email/plugin.py

```diff
@@ -24,6 +25,12 @@ class EmailRegistration(BasePlugin):
     def register_submission(self, submission: Submission, options: dict) -> None:
         submitted_data = submission.get_merged_data()
 
+        # explicitly get a reference
+        set_submission_reference(submission)
+
+        subject = _("[Open Forms] {} - submission {}").format(
```

Comment:
Can you please use named formatting? This makes translation easier :)

I saw a few cases but just mentioning this instance